pom.xml

```diff
@@ -3,284 +3,386 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.eharmony</groupId>
-    <artifactId>spotz</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
-    <packaging>jar</packaging>
-    <name>Spark HyperParameter Optimization</name>
+    <artifactId>spotz-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Spark HyperParameter Optimization Parent Pom</name>
     <description>A library for performing hyperparameter optimization on top of Spark</description>
+    <inceptionYear>2016</inceptionYear>
     <url>https://github.com/eHarmony/spotz.git</url>
 
+    <organization>
+        <name>eHarmony</name>
+        <url>http://www.eharmony.com</url>
+    </organization>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>eHarmony</id>
+            <url>https://repository.corp.eharmony.com/nexus/content/repositories/internal-snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <licenses>
+        <license>
```

Comment:
I haven't done any research on the licences available, so I'm not really sure of the ideal license for this project.  However, I'm open to suggestions.  What is the difference between the MIT license, Apache, and other common licenses?